### PR TITLE
* Update typing for js-captcha 1.3

### PR DESCRIPTION
**Pull request by PikachuEXE**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/robiveli/js-captcha/tree/1.3.0#options
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Ready to merge

File: types/js-captcha/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+}
+
+declare class jCaptcha {
+    constructor(options?: jCaptcha.Options);
+
+    validate(): void;
+    reset(): void;
+}
+
+export = jCaptcha;
```
Comment (peterblazejewicz) [inline types/js-captcha/index.d.ts:42]:
```suggestion
export as namespace jCaptcha;
export = jCaptcha;
```
I think this is an UMD module, used in the browser:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html
https://github.com/robiveli/js-captcha/blob/master/dist/js/index.js
https://unpkg.com/js-captcha@1.3.0/dist/js/index.js
(if accepted, please verify it works locally)

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
@PikachuEXE looks OK, can you review my comment about UMD?

**Comment by PikachuEXE** [comment]
@peterblazejewicz I think I missed one line when doing copy & paste
Fixed now

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@PikachuEXE thx for update!

**Pull request merged**